schema/rdl_schema_0.1.json

```
@@ -116,6 +122,16 @@
       },
       "minItems": 1,
       "uniqueItems": true
+    },
+    "sources": {
+      "title": "Sources",
+      "type": "array",
+      "description": "Related resources that serve as sources of information or predictions that contribute to the creation of the dataset.",
```

Since we aren't reusing the `RelatedResource` definition and to avoid confusion with `Resource`:

```suggestion
      "description": "Sources of information or predictions that contribute to the creation of the dataset.",
```